### PR TITLE
feat: test with non-helm deployment 

### LINKED ISSUE #342: test non-helm deployment in CI
**Describe the solution you'd like**
i think we are only testing helm deployment in CI. We should test `make deploy` or deployment from `manifest_staging` also

### PR DESCRIPTION
**Pull request by ashnamehrotra**
**What this PR does / why we need it**:
Creates a deployment with manifest_staging/deploy/eraser.yaml to test collect-scan-erase pipeline.

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #342

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by sozercan** [comment]
@ashnamehrotra is this a new test? (scenario we haven't covered before)

**Comment by ashnamehrotra** [comment]
> @ashnamehrotra is this a new test? (scenario we haven't covered before)

Its the same as the collector pipeline test checking if alpine is removed

**Comment by sozercan** [comment]
if it's a duplicate test, would it make sense to replace that test with this so we don't have a duplicate? we can add to description that this is non-helm too

File: test/e2e/tests/collector_pipeline/main_test.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func TestMain(m *testing.M) {
 		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.ScannerImage),
 		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.CollectorImage),
 		envfuncs.LoadDockerImageToCluster(util.KindClusterName, util.VulnerableImage),
-		util.DeployEraserManifest(util.EraserNamespace),
+		util.DeployEraserNonHelm(util.EraserNamespace, "../../../../manifest_staging/deploy", "eraser.yaml"),
```
Comment (sozercan) [inline test/e2e/tests/collector_pipeline/main_test.go]: should we call this `DeployEraserManifest` and the other one `DeployEraserHelm`

File: test/e2e/tests/collector_pipeline/eraser_test.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ import (
 )
 
 func TestCollectScanErasePipeline(t *testing.T) {
-	collectScanErasePipelineFeat := features.New("ImageCollector should run automatically, trigger the scanner, then the eraser pods").
+	collectScanErasePipelineFeat := features.New("Collector pods should run automatically, trigger the scanner, then the eraser pods. Non-helm deployment test.").
```
Comment (sozercan) [inline test/e2e/tests/collector_pipeline/eraser_test.go]:
```suggestion
	collectScanErasePipelineFeat := features.New("Collector pods should run automatically, trigger the scanner, then the eraser pods. Manifest deployment test.").
```

File: test/e2e/tests/collector_pipeline/main_test.go (hunk 1 above)
Comment (sozercan) [inline test/e2e/tests/collector_pipeline/main_test.go]: i think we have `util.ProviderResource` const for "eraser.yaml" (looks like unused?)

Comment (sozercan) [inline test/e2e/tests/collector_pipeline/main_test.go]: let's make `manifest_staging/deploy` as a const too

**Review by sozercan — APPROVED** [review]
lgtm, thank you!

**Pull request merged**